xml/System.Net/IPEndPoint.xml

```diff
@@ -793,10 +815,10 @@
         <Parameter Name="result" Type="System.Net.IPEndPoint" RefType="out" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">The IP endpoint to validate.</param>
+        <param name="result">The <see cref="T:System.Net.IPEndPoint" /> version of <paramref name="s" />.</param>
+        <summary>Determines whether <paramref name="s" /> is a valid IP endpoint.</summary>
+        <returns><see langword="true" /> if <paramref name="s" /> was able to be parsed as an IP endpoint; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if <paramref name="s" /> can be parsed as an IP endpoint; otherwise, <see langword="false" />.</returns>
```